conda/environments/all_cuda-120_arch-x86_64.yaml

```diff
@@ -0,0 +1,50 @@
+# This file is generated by `rapids-dependency-file-generator`.
+# To make changes, edit ../../dependencies.yaml and run `rapids-dependency-file-generator`.
+channels:
+- rapidsai
+- rapidsai-nightly
+- conda-forge
+- nvidia
+dependencies:
+- GPUtil>=1.4.0
+- c-compiler
+- click
+- cmake>=3.23.1,!=3.25.0
+- cuda-cudart-dev=12.0
+- cuda-nvcc_linux-64=12.0
```

Comment:
Generally we have been pinning cuda-version rather than the individual CTK packages.

Reply:
Also applies to libcufile-dev below.